lib/ui/src/components/sidebar/SidebarSubheading.stories.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 
 import SidebarSubheading from './SidebarSubheading';
 
-storiesOf('UI|Sidebar/SidebarSubheading', module)
-  .addParameters({
-    component: SidebarSubheading,
-  })
-  .add('default', () => <SidebarSubheading>Subheading</SidebarSubheading>);
+export default {
+  Component: SidebarSubheading,
```

Comment:
Component => component

Reply:
there's loads of other components stories that have `Component`?

Reply:
Well then they should probably be fixed 😂

Reply:
ok